docs/README.md

```diff
@@ -1,93 +1,96 @@
-# Sermant 开发和使用介绍
+# Introduction to Sermant Development and Usage
 
-Sermant 基于Java的字节码增强技术，通过 JavaAgent 对宿主应用进行非侵入式增强，以解决Java应用的微服务治理问题。Sermant的初衷是建立一个面向微服务治理的对开发态无侵入的解决方案生态，降低服务治理开发和使用的难度，通过抽象接口、功能整合、插件隔离等手段，达到简化开发、功能即插即用的效果。
+[简体中文](README-zh.md) | [English](README.md) 
 
-本文档对如何开发和使用 Sermant 做详细介绍。
+Sermant is a bytecode enhancement technology based on Java Agent. It uses Java Agent to enhance the host application in a non-intrusive way to solve the microservice governance problem of Java applications. The original intention of Sermant is to establish a solution ecosystem for micro-service governance that is non-intrusive to the development state, reduce the difficulty of service governance development and use, and achieve the effect of simplified development and plug and play by means of abstract interface, function integration and plug-in isolation.
```

Comment:
the reason why replace "plugin" by "plug-in"

Reply:
Resolved